core/alert.py

```diff
@@ -24,125 +22,29 @@ def is_not_run_from_api():
-                    )
-                    if langs != "readme.md" and langs.rsplit("_")[1].rsplit(".")[0] != ""
-                ]
-            )
-        )
-    # Importing messages
-    try:
-        msgs = getattr(
-            __import__(
-                "lib.language.messages_{0}".format(language),
-                fromlist=["all_messages"]
-            ),
-            "all_messages"
-        )()[str(msg_id)]
-    except Exception:
-        msgs = getattr(
-            __import__(
-                "lib.language.messages_en",
-                fromlist=["all_messages"]
-            ),
-            "all_messages"
-        )()[str(msg_id)]
-    if version() == 2:
-        return msgs.decode("utf8")
-    return msgs
-
-
-def input_msg(content):
-    """
-    build the input message to get input from users
-
-    Args:
-        content: content of the message
-
-    Returns:
-        the message in input structure
-    """
-    if version() == 2:
-        return color.color_cmd("yellow") + \
-               "[+] " + \
-               color.color_cmd("green") + \
-               content.encode("utf8") + \
-               color.color_cmd("reset")
-    else:
-        return bytes(
-            color.color_cmd("yellow") +
-            "[+] " + color.color_cmd("green") +
-            content +
-            color.color_cmd("reset"),
-            "utf8"
-        )
-
-
-def info(content, log_in_file=None, mode=None,
-         event=None, language=None, thread_tmp_filename=None):
+def info(content):
     """
     build the info message, log the message in
     database if requested, rewrite the thread temporary file
 
     Args:
         content: content of the message
-        log_in_file: log filename name
-        mode: write mode, [w, w+, wb, a, ab, ...]
-        event: standard event in JSON structure
-        language: the language
-        thread_tmp_filename: thread temporary filename
 
     Returns:
         None
     """
-    logger.info(content)
-    if is_not_run_from_api():  # prevent to stdout if run from API
-        if version() == 2:
-            sys.stdout.write(
-                color.color_cmd("yellow") +
-                "[+] [{0}] ".format(now()) +
-                color.color_cmd("green") +
-                content.encode("utf8") +
-                color.color_cmd("reset") +
-                "\n"
-            )
-        else:
-            sys.stdout.buffer.write(
-                bytes(
-                    color.color_cmd("yellow") +
-                    "[+] [{0}] ".format(now()) +
-                    color.color_cmd("green") +
-                    content +
-                    color.color_cmd("reset") +
-                    "\n",
-                    "utf8"
-                )
-            )
-            sys.stdout.flush()
+    sys.stdout.buffer.write(
+        bytes(
+            color.color_cmd("yellow")
+            + "[+] [{0}] ".format(now())
+            + color.color_cmd("green")
+            + content
```

Comment:
this would be perfect if we are just supporting python3, then I guess we can remove version function too from compatible

Reply:
it's already only python3...